fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsMemos.ts

```diff
@@ -32,7 +32,7 @@ export class FwsMemos {
       flightPhaseInhib: [],
```

Comment:
```suggestion
      flightPhaseInhib: [3, 4, 5, 6, 7, 8, 9, 10],
```
spotted one thing, if you look at the references this memo is suppressed for flight phases 3-10. Also, please rename the key of this entry to the same key as the first memo (280000009).